Parking/meter_parking.csv

```diff
@@ -0,0 +1,7 @@
+name,description,note,latitude,longitude
+,Optometry Building (OPT),,43.47584,-80.54485

```

Comment:
:key: :musical_note: I disagree. We should use meter_parking because that's what it is called: https://uwaterloo.ca/parking/visitors